Description: The version of python-sphinx in the archive (outside experimental)
right now is old, and isn't understanding the "Sphinx inventory version 2" now
being used at http://docs.python.org/dev/objects.inv. So none of the cross
references meant to point to docs.python.org were working.

Author: John Sullivan <john@wjsullivan.net>
Last-Update: 2010-09-06
Index: python-gevent/doc/conf.py
===================================================================
--- python-gevent.orig/doc/conf.py	2010-09-06 20:42:36.000000000 +0200
+++ python-gevent/doc/conf.py	2010-09-06 20:42:49.000000000 +0200
@@ -35,7 +35,7 @@
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.coverage', 'sphinx.ext.intersphinx', 'mysphinxext']
 
-intersphinx_mapping = {'http://docs.python.org/dev': None}
+intersphinx_mapping = {'http://docs.python.org/': None}
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
